logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt

```diff
@@ -42,26 +44,30 @@ import kotlinx.datetime.Instant
 
 // TODO: look into excluding self clients from sendConfirmation or run sendLastReadMessageToOtherClients if
 //  the conversation does not need to be notified
+@Suppress("LongParameterList")
 class UpdateConversationReadDateUseCase internal constructor(
     private val conversationRepository: ConversationRepository,
     private val messageSender: MessageSender,
     private val currentClientIdProvider: CurrentClientIdProvider,
     private val selfUserId: UserId,
     private val selfConversationIdProvider: SelfConversationIdProvider,
     private val sendConfirmation: SendConfirmationUseCase,
+    private val scope: CoroutineScope
 ) {
 
     /**
      * @param conversationId The conversation id to update the last read date.
      * @param time The last read date to update.
      */
     suspend operator fun invoke(conversationId: QualifiedID, time: Instant) {
```

Comment:
maybe remove the suspend modifier here since it is not needed anymore

Reply:
Good catch! Did a test run and all good 👌🏻